lms/djangoapps/support/static/support/js/views/enrollment.js

```diff
@@ -39,12 +39,7 @@
                     user: user,
                     enrollments: this.enrollments,
                     formatDate: function(date) {
-                        if (!date) {
-                            return 'N/A';
-                        }
-                        else {
-                            return moment(date).format('MM/DD/YYYY (H:MM UTC)');
-                        }
+                        return date ? moment.utc(date).format('lll z') : 'N/A';
```

Comment:
Reporter Said the issue is resolved on the sandbox.
See the comment: https://openedx.atlassian.net/browse/EDUCATOR-496?focusedCommentId=266878&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-266878